### PR TITLE
React1 homework limbo

### PR DESCRIPTION
**Pull request by mag-da-lenka**

### CODE REVIEW

File: react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+class TodoList extends React.Component {
+  render() {
+    const toDoListMapped = todoThings.map((tasks) => {
+      return (
+        <div key={tasks.nr}>
+          {tasks.nr}. {tasks.date}: {tasks.task}
+        </div>
+      );
+    });
+    return <main className="todolist">
```
Comment (fernandoabolafio) [inline react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js]: This is a list, so you should use the appropriate HTML tags https://www.w3schools.com/html/html_lists.asp

File: react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      task: 'Listen to Sepultura',
+      date: myDateFormat_month_number4
+    }
+  ];
+
+class TodoList extends React.Component {
+  render() {
+    const toDoListMapped = todoThings.map((tasks) => {
+      return (
+        <div key={tasks.nr}>
```
Comment (fernandoabolafio) [inline react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js]: You could break it down more and add a component for the todo item (e.g <Todo />)

File: react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js (hunk 1 above)
Comment (mag-da-lenka) [reply react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js]: Thank you. Done.

File: react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js (hunk 2 above)
Comment (mag-da-lenka) [reply react/week1/react1-hw-todo-app/react1-hw-todo-app/src/index.js]: Thank you.

**Comment by mag-da-lenka** [comment]
Hi. Thank you for your feedback. I have implemented some changes. You will see the commits in a few minutes :)

**Comment by mag-da-lenka** [comment]
I still feel a little confused but I hope I have managed to improve :)

**Pull request merged**